Template/Tag/MatomoTag.web.js

```diff
@@ -210,6 +210,10 @@
             // to initialize the tracker during window.piwikPluginAsyncInit
 
             var matomoConfig = parameters.get('matomoConfig', {});
+            if (matomoConfig.useExternalTracker) {
```

Comment:
This one I'm not quite sure I understand?

Reply:
Thank you for your feedback. I have merged this setting with the "bundleTracker" setting into a new "trackerSource" setting. 
trackerSource=external is for websites that already load matomo.js (asynchronously) and do not require the js tracker to be loaded by the container, while the container cannot detect it is already being loaded because it is loaded async.

Reply:
Because the "bundleTracker" setting gets removed with this change, already existing configs will default back to  trackerSource=bundle, if you do not like this, we can change it